dotnet/src/VectorDataIntegrationTests/Directory.Build.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <Import Project="../../Directory.Build.props" />
```

Comment:
It's great that we have a props file, so we can move all the things that would be common for all test projects here 👍

Reply:
Yep. There's some opportunity to refactor common stuff, but I didn't want to go too far in this already-huge PR.